assets/sass/components/global/_googlesitekit-google-tag-gateway-toggle.scss

```diff
@@ -32,6 +32,14 @@
 			}
 		}
 
+		.googlesitekit-notice--new {
```

Comment:
Are the styles unique to this notice or is this relevant to all New notices? Since this is just sizing, it's odd that it would be so specific here. Is it even needed at all? If so, why not all notices?

Reply:
The change related to the sizing has been removed as that change currently already exists for such notices. I've only left the style for the spacing. Thank you!